### PR TITLE
Account Lookup

### PR DESCRIPTION
**Pull request by jfantunes**
PR for the account lookup.
Since we started the task on the main branch and only switched to dev recently, all code is not included here. So it's better to check the main branch and add if needed, a comment here. 
Unit tests added to the domain, Integration tests to the infrastructure, and the unit tests are in progress for the service and will be concluded tomorrow.

### CODE REVIEW

File: modules/account-lookup-svc/src/index.ts [hunk 1]
```diff
@@ -19,11 +19,9 @@
  their names indented and be marked with a '-'. Email address can be added
  optionally within square brackets <email>.
 
- * Gates Foundation
- - Name Surname <name.surname@gatesfoundation.com>
-
- * Coil
- - Jason Bruwer <jason.bruwer@coil.com>
+ * Arg Software
```
Comment (elnyry-sam-k) [inline modules/account-lookup-svc/src/index.ts:22]: why delete previous contributors? please append..

File: modules/client/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# LICENSE
```
Comment (elnyry-sam-k) [inline modules/client/LICENSE.md:1]: Please use Mojaloop license: https://docs.mojaloop.io/getting-started/license.html

File: modules/domain/src/index.ts [hunk 3]
```diff
@@ -19,11 +19,9 @@
  their names indented and be marked with a '-'. Email address can be added
  optionally within square brackets <email>.
 
- * Gates Foundation
- - Name Surname <name.surname@gatesfoundation.com>
-
- * Coil
- - Jason Bruwer <jason.bruwer@coil.com>
+ * Arg Software
```
Comment (elnyry-sam-k) [inline modules/domain/src/index.ts:22]: Same here, please address these in all the places.. Thank you!

File: modules/account-lookup-svc/src/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jfantunes) [reply modules/account-lookup-svc/src/index.ts:33]: My bad. Updated!

File: modules/client/LICENSE.md (hunk 2 above)
Comment (jfantunes) [reply modules/client/LICENSE.md:1]: Updated!

Comment (elnyry-sam-k) [reply modules/client/LICENSE.md:1]: Thanks! please use this license file as well: https://github.com/mojaloop/documentation/blob/master/LICENSE.md

Comment (jfantunes) [reply modules/client/LICENSE.md:1]: @elnyry-sam-k, so it should be like this:

image

Or it is to replace the previous with this one, like this:
image

File: modules/account-lookup-svc/src/index.ts (hunk 1 above)
Comment (jfantunes) [reply modules/account-lookup-svc/src/index.ts:33]: Updated! My bad! Thanks for notifying 👌

File: modules/domain/src/index.ts (hunk 3 above)
Comment (jfantunes) [reply modules/domain/src/index.ts:33]: Done!

File: modules/client/LICENSE.md (hunk 2 above)
Comment (elnyry-sam-k) [reply modules/client/LICENSE.md:1]: yes second one please.. it must now be Mojaloop Foundation's license

File: modules/domain/src/index.ts (hunk 3 above)
Comment (pedrosousabarreto) [reply modules/domain/src/index.ts:33]: @elnyry-sam-k and @jfantunes  There is no previous contributor, this a new repo.

**Review by pedrosousabarreto — APPROVED** [review]
Looks good!
Good work on the your first PR, well done.

**Comment by pedrosousabarreto** [comment]
Approved. 
As per tradition, developers get to merge the PR and delete the previous branch after merge with main.

**Pull request merged**